tests/console/systemd_nspawn.pm

```diff
@@ -70,9 +70,11 @@ sub run {
     assert_script_run 'machinectl shell messagebus@' . $machine . ' /usr/bin/whoami | grep messagebus';
     assert_script_run "machinectl shell $machine /usr/bin/systemctl status systemd-journald | grep -B100 -A100 'active (running)'";
     assert_script_run "machinectl stop test1";
-    script_retry 'systemctl status systemd-nspawn@' . $machine . ' | grep "Reached target Power-Off"', retry => 30, delay => 5;
+    script_retry 'systemctl status systemd-nspawn@' . $machine, retry => 30, delay => 5, expect => 3;
+    script_retry 'journalctl -n10 -u systemd-nspawn@' . $machine . ' | grep "Reached target Power-Off"', retry => 30, delay => 5;
```

Comment:
Wouldn't it be better to use validate_script_output?
It would be easier to review what went wrong by seeing the command output and will not be affected by pipefail.

Reply:
Updated to use `validate_script_output`: http://kazhua.qa.suse.de/tests/244#step/systemd_nspawn/107